src/peerbook/libp2p_peerbook.erl

```diff
@@ -107,14 +126,20 @@ init([TID, SigFun]) ->
     Group = group_create(SwarmName),
     Opts = libp2p_swarm:opts(TID),
     StaleTime = libp2p_config:get_opt(Opts, [?MODULE, stale_time], ?DEFAULT_STALE_TIME),
+    GossipGroup = case (catch libp2p_swarm:gossip_group(TID)) of
+                      {'EXIT', _} -> undefined;
```

Comment:
Why?

Reply:
Ah, this was a hack to be able to test the peerbook without a gossip group.. There are a number of tests in the peerbook_SUITE that construct a peerbook outside of a swarm, which this makes a bit easier

Reply:
Added a comment to help clarify